submissions/april-1st/submission-april-1st.md

```diff
@@ -0,0 +1,208 @@
+
+(disco gif: easy)
+
+
+
+**Shrek**
+
+by [jerzygorski](https://mods.factorio.com/user/jerzygorski)
+
+He's back and he means business! [This mod](https://mods.factorio.com/mod/shrek) retextures your character as Shrek, and proves that he can do rocket science just like how he can rescue the princess! He mines with his fists and has his own armor and weapon texture too, perfect for getting those pesky bugs out of your swamp planet!
+
+(mining gif: easy) (armor + shooting gif: easy)
+
+
+
+**Bumble Bots**
+
+by [Maxreader](https://mods.factorio.com/user/Maxreader)
+
+If you ever thought about your bots doing everything for you as your little worker bees, [here is a mod](https://mods.factorio.com/mod/bumble-bots) that makes it literal! The mod retextures and resounds combat robots, construction and logistic robots, roboports and logistic chests. The organic structures (especially flowers) among the machines makes an interesing mix.
+
+(logisctic gif: easy) 
+
+(combat gif: easy)
+
+
+
+**Sushi Mod**
+
+by [Xaetral](https://mods.factorio.com/user/Xaetral)
+
+Sushi belt fans, [this mod](https://mods.factorio.com/mod/sushi) gives you the means to truly flex your skills! It also helps if you are tired of eating only raw fish! The menu has 6 sushi and 6 maki types, each crafted from one fish. Bon appetit!
+
+(gif from mod / image from mod page)
+
+
+
+**1000 Sounds to die for**
+
+by [vralfy](https://mods.factorio.com/user/vralfy)
+
+[This sound mod](https://mods.factorio.com/mod/1000sounds) allows you to customize a number of sounds, in particular character deaths. Yes, death in Factorio is usually a tragedy, but you can make it more comedic! Got hit by a train? Hear it honk! Killed by another player? Why not set that to the classic [Wilhelm Scream](https://www.youtube.com/watch?v=r6JK-gRELI0&ab_channel=GamingSoundFX)? You can also customize sounds for a few other events like starting/completing a research, launching a rocket, or players entering/exiting a game. The sound library of this mod borrows from Doom and Quake among some other places. If you like this mod, along the same lines is one for [pain sounds](https://mods.factorio.com/mod/pain), from Doom or custom sources.
+
+(GUI image: easy)
+
+
+
+**Choleric cars**
+
+by [Therenas](https://mods.factorio.com/user/Therenas)
+
+Did your car/tank just crash into rock? Yeah it sucks. [This sound mod](https://mods.factorio.com/mod/cholericcars) lets the engineer express his opinion about it very clearly (sometimes with expletives). It definitely captures the feeling of the moment, often comedically well!
```

Comment:
```suggestion
Did your car/tank just crash into rock? Yeah it sucks. [This sound mod](https://mods.factorio.com/mod/cholericcars) lets the engineer express his opinion about it very clearly (sometimes with expletives). It definitely captures the feeling of the moment, often comedically well! And we promise this mod isn't showcased here because the author is our Editor-In-Chief. We simply assume it's his stress outlet for dealing with the rest of us.
```